depends/packages/miniupnpc.mk

```diff
@@ -1,14 +1,15 @@
 package=miniupnpc
-$(package)_version=2.0.20170509
+$(package)_version=2.0.20180203
 $(package)_download_path=https://miniupnp.tuxfamily.org/files
 $(package)_file_name=$(package)-$($(package)_version).tar.gz
-$(package)_sha256_hash=d3c368627f5cdfb66d3ebd64ca39ba54d6ff14a61966dbecb8dd296b7039f16a
+$(package)_sha256_hash=90dda8c7563ca6cd4a83e23b3c66dbbea89603a1675bfdb852897c2c9cc220b7
 
 define $(package)_set_vars
 $(package)_build_opts=CC="$($(package)_cc)"
 $(package)_build_opts_darwin=OS=Darwin LIBTOOL="$($(package)_libtool)"
 $(package)_build_opts_mingw32=-f Makefile.mingw
 $(package)_build_env+=CFLAGS="$($(package)_cflags) $($(package)_cppflags)" AR="$($(package)_ar)"
+$(package)_build_env+=CFLAGS=-D_DARWIN_C_SOURCE
 endef
 
 define $(package)_preprocess_cmds
```

Comment:
missing changes in native_ccache.mk

Reply:
@PastaPastaPasta Dash doesn't have this package and was also removed from bitcoin later